src/main/java/gregtech/api/block/VariantActiveBlock.java

```diff
@@ -47,22 +95,39 @@ protected BlockStateContainer createBlockState() {
         Class<T> enumClass = GTUtility.getActualTypeParameter(getClass(), VariantActiveBlock.class, 0);
         this.VARIANT = PropertyEnum.create("variant", enumClass);
         this.VALUES = enumClass.getEnumConstants();
-        return new BlockStateContainer(this, VARIANT, ACTIVE);
+        return new ExtendedBlockState(this, new IProperty[]{VARIANT, ACTIVE_DEPRECATED}, new IUnlistedProperty[]{ACTIVE});
     }
 
     @Override
-    public int damageDropped(@Nonnull IBlockState state) {
-        return getMetaFromState(state) - (state.getValue(ACTIVE) ? 8 : 0);
+    public IExtendedBlockState getExtendedState(IBlockState state, IBlockAccess world, BlockPos pos) {
+        IExtendedBlockState ext = (IExtendedBlockState) state;
+        ACTIVE_BLOCKS.putIfAbsent(Minecraft.getMinecraft().world.provider.getDimension(), new ObjectOpenHashSet<>());
```

Comment:
Getting a crash on the latest commit when loading into the world with a running Large Bronze Boiler because `Minecraft.getMinecraft.world` is null